### PR TITLE
Add context to wiki links

### PR DESCRIPTION
**Pull request by kbjarkefur**

### CODE REVIEW

File: chapters/4-acquisition.tex [hunk 1]
```diff
@@ -331,29 +350,29 @@ \subsection{Developing a data collection instrument}
...
 and gather qualitative information relevant to any of the planned survey modules.
 A pre-pilot is particularly important when designing new survey instruments.
 
-The second step is a \textbf{content-focused pilot}\sidenote{
-  \url{https://dimewiki.worldbank.org/Piloting\_Survey\_Content}}.
+The second step is a \textbf{content-focused pilot}.
 The objectives at this stage are to improve the structure and length of the questionnaire,
 refine the phrasing and translation of specific questions,
 and confirm coded response options are exhaustive.\sidenote{
+  A checklist for content-focused pilots can be found on the DIME Wiki:
```
Comment (mariaruth) [inline chapters/4-acquisition.tex:373]: Note that @Avnish95 is changing the name of this page to Checklist: Content-focused Pilot

File: chapters/4-acquisition.tex [hunk 2]
```diff
@@ -331,29 +350,29 @@ \subsection{Developing a data collection instrument}
...
+  More details on how to structure survey pilots,
+	the different components of comprehensive survey pilot and
+	link to many other resources can be found on the DIME Wiki:
+ \url{https://dimewiki.worldbank.org/Survey\_Pilot}}\index{survey pilot}
 of the questionnaire.
-A comprehensive survey pilot is a critical stage in survey design.\sidenote{
-	\url{https://dimewiki.worldbank.org/Survey\_Pilot}}
+A comprehensive survey pilot is a critical stage in survey design.
 The pilot must be done out-of-sample,
 but in a context as similar as possible to the study sample.
```
Comment (mariaruth) [inline chapters/4-acquisition.tex:360]: add link to https://dimewiki.worldbank.org/wiki/Survey_Pilot_Participants here

File: chapters/4-acquisition.tex [hunk 3]
```diff
@@ -331,29 +350,29 @@ \subsection{Developing a data collection instrument}
...
-  \url{https://dimewiki.worldbank.org/Piloting\_Survey\_Content}}.
+The second step is a \textbf{content-focused pilot}.
 The objectives at this stage are to improve the structure and length of the questionnaire,
 refine the phrasing and translation of specific questions,
 and confirm coded response options are exhaustive.\sidenote{
+  A checklist for content-focused pilots can be found on the DIME Wiki:
 	\url{https://dimewiki.worldbank.org/index.php?title=Checklist:\_Refine\_the\_Questionnaire\_(Content)}}
-In addition, it is an opportunity to test and refine all survey protocols,\sidenote{
-	\url{https://dimewiki.worldbank.org/Piloting\_Survey_Protocols}}
+In addition, it is an opportunity to test and refine all survey protocols,
```
Comment (mariaruth) [inline chapters/4-acquisition.tex:375]: Add here: "For a detailed discussion of testing survey protocols, go to https://dimewiki.worldbank.org/wiki/Piloting_Survey_Protocols" BUT @Avnish95 is changing the name so once that is done will be Checklist: Piloting Survey Protocols

File: chapters/4-acquisition.tex [hunk 4]
```diff
@@ -499,6 +524,8 @@ \subsection{Using electronic survey features}
 
 The final stage of survey piloting, the data-focused pilot, should be done at this stage (after the questionnaire is programmed).
 The objective of this \textbf{data-focused pilot}\sidenote{
+  A checklist with best practices important to remember during a data-focused pilot
```
Comment (mariaruth) [inline chapters/4-acquisition.tex:527]: Note that @Avnish95 is changing the name of this page, need to update link once that's done

File: chapters/4-acquisition.tex [hunk 5]
```diff
@@ -331,29 +350,29 @@ \subsection{Developing a data collection instrument}
...
 At this point, it is useful to do a \textbf{survey pilot}\sidenote{
- \url{https://dimewiki.worldbank.org/Piloting\_Survey\_Content}}\index{survey pilot}
+  More details on how to structure survey pilots,
+	the different components of comprehensive survey pilot and
+	link to many other resources can be found on the DIME Wiki:
+ \url{https://dimewiki.worldbank.org/Survey\_Pilot}}\index{survey pilot}
 of the questionnaire.
-A comprehensive survey pilot is a critical stage in survey design.\sidenote{
-	\url{https://dimewiki.worldbank.org/Survey\_Pilot}}
+A comprehensive survey pilot is a critical stage in survey design.
```
Comment (mariaruth) [inline chapters/4-acquisition.tex:358]:
```suggestion
A checklist for how to prepare for a survey pilot can be found on the DIME Wiki: https://dimewiki.worldbank.org/wiki/Checklist:_Preparing_for_a_Survey_Pilot.
A comprehensive survey pilot is a critical stage in survey design.
```

File: chapters/4-acquisition.tex (hunk 4 above)
Comment (kbjarkefur) [reply chapters/4-acquisition.tex:534]: done in 04a6a926ec8986149a58e7f5e0f4dc8848a1dc3a

File: chapters/4-acquisition.tex (hunk 3 above)
Comment (kbjarkefur) [reply chapters/4-acquisition.tex:379]: done in 04a6a926ec8986149a58e7f5e0f4dc8848a1dc3a

File: chapters/4-acquisition.tex (hunk 2 above)
Comment (kbjarkefur) [reply chapters/4-acquisition.tex]: done in 04a6a926ec8986149a58e7f5e0f4dc8848a1dc3a

File: chapters/4-acquisition.tex (hunk 1 above)
Comment (kbjarkefur) [reply chapters/4-acquisition.tex:377]: done in 04a6a926ec8986149a58e7f5e0f4dc8848a1dc3a

**Pull request merged**